packages/relic/lib/src/extension/base64_extension.dart

```diff
@@ -0,0 +1,24 @@
+import 'dart:convert';
+
+/// Helper method to check if a string is valid Base64.
+///
+/// This method is used internally to validate that the Base64-encoded credentials
+/// are correctly formatted before attempting to decode them.
+extension Base64Extension on String {
+  bool get isValidBase64 {
+    final base64Pattern = RegExp(r'^[A-Za-z0-9+/]+={0,2}$');
```

Comment:
Question/Discussion: This pattern definition is different from the one used to identify base64 in auth. Which one is correct? And can the implementation be shared? `packages/serverpod_serialization/lib/src/auth_encodingd.dart`

Reply:
i removed this, we dont actually need it, because this `utf8.decode(base64Decode(this))` is gonna throw an exception if the encoding is invalid, so we dont actually need it to check first.